src/components/IOUConfirmationList.js

```diff
@@ -201,7 +210,12 @@ class IOUConfirmationList extends Component {
             <View style={[styles.flex1, styles.w100, styles.justifyContentBetween]}>
                 <View style={[styles.flex1]}>
                     <OptionsList
-                        listContainerStyles={[styles.flexGrow0]}
+                        listContainerStyles={[{
+                            // Give max height to the list container so that it does not extend
+                            // beyond the comment view as well as button
+                            maxHeight: Dimensions.get('window').height - MINIMUM_BOTTOM_OFFSET
```

Comment:
Any reason to use `Dimensions.get` here instead of updating this component to use `withWindowDimensions`?

Reply:
Didn't realise we had `withWindowDimensions`. Made the change now!